README.md

```diff
@@ -1,5 +1,10 @@
 Apache Kafka
 =================
+
+![CI](https://github.com/apache/kafka/actions/workflows/ci.yml/badge.svg?branch=trunk&event=push)
+
```

Comment:
Could you please consider removing the new line? 
![Screenshot From 2024-12-14 10-42-04](https://github.com/user-attachments/assets/27d9a3d3-df23-4865-935f-e12222b8b486)